pkg/kvstore/client.go

```diff
@@ -30,7 +30,7 @@ func initClient(module backendModule, opts *ExtraOptions) error {
 	c, errChan := module.newClient(opts)
 	if c == nil {
 		err := <-errChan
-		log.WithError(err).Fatalf("Unable to create etcd client")
+		log.WithError(err).Fatalf("Unable to create kvstore %s client", module.getName())
```

Comment:
Nit: use `WithFields` here for a generic log message with the field being the output of `module.getName()`.